osquery/killswitch/plugins/killswitch_json.cpp

```diff
@@ -0,0 +1,43 @@
+#include <string>
+
+#include <osquery/killswitch/killswitch_plugin.h>
+
+#include "osquery/core/json.h"
+#include "osquery/killswitch/plugins/killswitch_json.h"
+
+namespace osquery {
+
+Status KillswitchJSON::refresh() {
+  std::string content;
+
+  auto status = getJSON(content);
+  if (!status.ok()) {
+    return status;
+  }
+
+  auto doc = JSON::newObject();
+  if (!doc.fromString(content) || !doc.doc().IsObject()) {
+    return Status(1, "Error parsing the config JSON. Content : " + content);
+  }
+  clearCache();
+
+  for (auto& m : doc.doc().GetObject()) {
```

Comment:
It is a bit vague what name `m` means here. May I ask you to change it to something more clear?